substrate/client/network/src/service.rs

```diff
@@ -743,6 +743,21 @@ impl<B: BlockT + 'static, H: ExHashT> NetworkService<B, H> {
 		rx.await.map_err(|_| ())
 	}
 
+	/// Returns a collection of currently connected (open) peers.
+	pub async fn open_peers(&self) -> Result<Vec<PeerId>, ()> {
```

Comment:
The name is strange, this would make way more sense to me:
```suggestion
	pub async fn connected_peers(&self) -> Result<Vec<PeerId>, ()> {
```